src/registryctl/config/registry/config.go

```diff
@@ -0,0 +1,34 @@
+package registry
+
+import (
+	"fmt"
+	"github.com/docker/distribution/configuration"
+	storagedriver "github.com/docker/distribution/registry/storage/driver"
+	"os"
+)
+
+// DefaultRegConf ...
+const DefaultRegConf = "/etc/registry/config.yml"
+
+// StorageDriver ...
+var StorageDriver storagedriver.StorageDriver
+
+// ResolveConfiguration ...
+func ResolveConfiguration(configPath ...string) (*configuration.Configuration, error) {
```

Comment:
I imagine the path of configuration file should be a parm to the binary? so the parm of this func is must-have.

and please try not to write comment like `...`

Reply:
updated.